library/build.gradle

```diff
@@ -2,34 +2,31 @@ import io.gitlab.arturbosch.detekt.Detekt
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 buildscript {
-  apply from: "gradle/dependencies.gradle"
-
   repositories {
     mavenCentral()
     gradlePluginPortal()
     google()
   }
 
   dependencies {
-    classpath libs.android.gradlePlugin.project
-    classpath libs.kotlin.gradlePlugin.api
-    classpath libs.mavenPublish.gradlePlugin
-    classpath libs.dokka.gradlePlugin
-    classpath libs.detekt.gradlePlugin
-    classpath libs.gradleVersions.gradlePlugin
-    classpath libs.wireGradlePlugin
-    classpath libs.kotlin.x.binaryCompatibility.gradlePlugin
+    classpath(libs.android.gradlePlugin)
```

Comment:
leftovers from migrating to kts, will restore previous version (remove braces) or migrate it properly